src/mavsdk/plugins/follow_me/follow_me_impl.cpp

```diff
@@ -219,7 +258,7 @@ bool FollowMeImpl::is_config_ok(const FollowMe::Config& config) const
     if (config.min_height_m < CONFIG_MIN_HEIGHT_M) {
         LogErr() << debug_str << "Err: Min height must be atleast 8.0 meters";
     } else if (config.follow_distance_m < CONFIG_MIN_FOLLOW_DIST_M) {
-        LogErr() << debug_str << "Err: Min Follow distance must be atleast 1.0 meter";
+        LogErr() << debug_str << "Err: Min Follow distance must be atleast 2.0 meter";
```

Comment:
Nitpick:

```suggestion
        LogErr() << debug_str << "Err: Min Follow distance must be at least 2.0 meter";
```

Reply:
BTW can we print `CONFIG_MIN_FOLLOW_DIST_M` here? Something like

```
LogErr() << debug_str << "Err: Min Follow distance must be at least " << CONFIG_MIN_FOLLOW_DIST_M << " meter";
```

Reply:
Changes applied!